tests/utility.py

```diff
@@ -26,9 +29,10 @@ def assert_json_file_equal(file1, file2):
 
 
 def get_docker_client():
-    envvars.load()
     utility = Utility(envvars, output)
+    envvars.load(force=True)
```

Comment:
why move after utility initialized? The env should be used by utility?

Reply:
It is for changing env variable "DEFAULT_PLATFORM"

Reply:
why not load first then use it for utility?

Reply:
updated code